client/wfprev-war/src/main/angular/src/app/components/edit-project/project-fiscals/project-fiscals.component.stories.ts

```diff
@@ -0,0 +1,105 @@
+import { moduleMetadata, type Meta, type StoryObj } from '@storybook/angular';
+import { MatTabsModule } from '@angular/material/tabs';
+import { MatButtonModule } from '@angular/material/button';
+import { MatSelectModule } from '@angular/material/select';
+import { MatSlideToggleModule } from '@angular/material/slide-toggle';
+import { MatExpansionModule } from '@angular/material/expansion';
+import { MatMenuModule } from '@angular/material/menu';
+import { ReactiveFormsModule, FormGroup, FormControl } from '@angular/forms';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { FiscalYearProjectsComponent } from 'src/app/components/edit-project/project-details/fiscal-year-projects/fiscal-year-projects.component';
+
+const meta: Meta<FiscalYearProjectsComponent> = {
+  title: 'FiscalYearProjectsComponent',
+  component: FiscalYearProjectsComponent,
+  tags: ['autodocs'],
+  decorators: [
+    moduleMetadata({
+      imports: [
+        FiscalYearProjectsComponent,
+        MatTabsModule,
+        MatButtonModule,
+        MatSelectModule,
+        MatSlideToggleModule,
+        MatExpansionModule,
+        MatMenuModule,
+        ReactiveFormsModule,
+        BrowserAnimationsModule,
+      ],
+    }),
+  ],
+};
+
+export default meta;
+type Story = StoryObj<FiscalYearProjectsComponent>;
+
+export const Default: Story = {
+  args: {
+    projectFiscals: [
+      { fiscalYearFormatted: '2023/2024' },
+      { fiscalYearFormatted: '2024/2025' }
+    ],
+    fiscalForms: [
+      new FormGroup({
+        fiscalYear: new FormControl('2023/2024'),
```

Comment:
Doesn't the converter function in tools use a format like `2025/26` ?

Reply:
Good catch! updated.